docs/fundamentals/app-host-overview.md

```diff
@@ -465,6 +488,37 @@ This logic can easily be inverted to connect to an existing Redis resource when
 > [!IMPORTANT]
 > .NET Aspire provides common APIs to control the modality of resource builders, allowing resources to behave differently based on the execution mode. The fluent APIs are prefixed with `RunAs*` and `PublishAs*`. The `RunAs*` APIs influence the local development (or run mode) behavior, whereas the `PublishAs*` APIs influence the publishing of the resource.
 
+## Resource relationships  
+
+Resource relationships link resources together. Relationships are informational and don't impact an app's runtime behavior. Instead, they're used when displaying details about resources in the dashboard. For example, relationships are visible in the [dashboard's resource details](./dashboard/explore.md#resource-details), and `Parent` relationships control resource nesting on the resources page.
+
+Relationships are automatically created by some app model APIs. For example:
+
+- <xref:Aspire.Hosting.ResourceBuilderExtensions.WithReference*> adds a relationship to the target resource with the type `Reference`.
+- <xref:Aspire.Hosting.ResourceBuilderExtensions.WaitFor*> adds a relationship to the target resource with the type `WaitFor`.
+- Adding a database to a DB container creates a relationship from the database to the container with the type `Parent`.
+
+Relationships can also be explicitly added to the app model using `WithRelationship` and `WithParentRelationship`.
```

Comment:
nit: WithRelationship and WithParentRelationship can be changed to an xref to API docs once they're available.